chatkit/src/main/java/com/stfalcon/chatkit/messages/MessageHolders.java

```diff
@@ -557,8 +557,10 @@ public void applyStyle(MessagesListStyle style) {
     }
 
     public static void applyTextTransformation(TextView textView, String text, int gravity, MessagesListStyle style){
-        List<MessageTextUtils.PatternDescriptor> textUrls = MessageTextUtils.Companion.getTextPatterns(text);
-        @SuppressLint("RtlHardcoded") SpannableString sText = MessageTextUtils.Companion.transform(text, textUrls,
+        List<EmojisDetect.EmojiDescriptor> emojisDescriptors = EmojisDetect.Companion.detectEmojis(text);
+        String emojiText = EmojisDetect.Companion.transformText(text,emojisDescriptors);
```

Comment:
rename it to text, we dont want to have multiple instances of same text with different names